<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>obnovlenie_1</class>
 <widget class="QDialog" name="obnovlenie_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Change</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:MediumSeaGreen</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ИЗМЕНЕНИЕ ДАННЫХ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Изменить Название</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_cahnge">
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Код поля</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="kod_change">
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Изменить Количество</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="kolvo_change">
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Изменить Производителя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="manuf_change">
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
